sj201_interface/led/animations.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2020 Mycroft AI Inc.
```

Comment:
There are more animations in Dinkum that should be added here

Reply:
Dinkum animations are NeoPixel-specific from other sources. Any additional animations should be added [in OPM](https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/88)